src/backend/sev/builder.rs

```diff
@@ -75,6 +75,17 @@ impl TryFrom<super::config::Config> for Builder {
                 .create_vm()
                 .context("Failed to create a virtual machine")?;
 
+            // Enable KVM_CAP_UNMAPPED_PRIVATE_MEM.
```

Comment:
What good does this comment add to the code? IMHO, its usefulness is questionable.